cdb2api/cdb2api.c

```diff
@@ -494,6 +487,10 @@ static void do_init_once(void)
     }
 }
 
+#define cdb2_skipws(str)                                                                                               \
```

Comment:
I think it's a common practice to wrap it in a `do {} while (0)` this macro so it can be executed in a if branch without `{}`.